build.assets/robotest_run_suite.sh

```diff
@@ -7,10 +7,13 @@ readonly UPGRADE_FROM_DIR=${1:-$(pwd)/../upgrade_from}
 declare -A UPGRADE_MAP
 
 # latest patch release on compatible LTS, keep this up to date
-UPGRADE_MAP[6.1.18]="ubuntu:18"
+UPGRADE_MAP[6.1.28]="ubuntu:18"
 
 # latest patch release on supported non-LTS version, keep this up to date
-UPGRADE_MAP[6.3.6]="ubuntu:18"
+UPGRADE_MAP[6.3.18]="ubuntu:18"
+
+# latest patch release on supported LTS version, keep this up to date
+UPGRADE_MAP[7.0.6]="ubuntu:18"
```

Comment:
Is 7.0 LTS?  It seems like this comment should be left out and 7.0.6 should fall under "supported non-LTS version" along with 6.3.